packages/mds-providers/index.ts

```diff
@@ -93,4 +93,33 @@ const providers: Readonly<{ [id: string]: Readonly<Provider> }> = Object.freeze(
   })
 })
 
-export = providers
+/**
+ * convert provider_id to provider name (if available)
+ * @param  {UUID} provider_id
```

Comment:
These type annotations are redundant thanks to TS, I'd recommend removing. Commented type annotations can fall out of sync with the codebase, harder for the TS types to!

Reply:
Good call